src/components/schemas/fields.yaml

```diff
@@ -239,6 +239,7 @@ spent_at:
   description: |
     It signifies when this expense was made.
     This date cannot be in future.
+    This field is not required for creating Per Diem Expenses
```

Comment:
changing this description here, will show it in each and every API, that is using `spent_at`, even CCC, Please check it out once, if yes, we'll have to somehow add the description only for expenses API `spent_at`.